src/routes/events/FeaturedEvent.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+    import Button from '$lib/components/Button.svelte';
+
+    export let tags: string[] = [];
+    export let schedule: string[] = [];
+</script>
+
+<!-- Featured Card Event
+tags and schedule - pass as props
+image, event name, description - pass as slot -->
+<div
+    class="flex w-full flex-col items-center justify-center gap-6 rounded-3xl bg-csi-neutral-50 p-4 text-center shadow-lg md:flex-row md:justify-normal md:gap-8 md:text-left dark:bg-csi-neutral-900"
+>
+    <div class="flex w-full flex-col border-b-2 md:w-1/3 md:flex-row md:border-b-0">
+        <slot name="image" />
+    </div>
+
+    <div class="flex w-2/3 flex-col items-center gap-6 md:items-start">
+        <!-- Tags -->
+        <div class="flex gap-4">
+            {#each tags as tag}
+                <Button>
+                    {tag}
+                </Button>
```

Comment:
Just a nitpick; the extra lines are kinda overkill. 😅
```suggestion
                <Button>{tag}</Button>
```